test/verify.tests.js

```diff
@@ -114,70 +114,97 @@ describe('verify', function() {
     });
 
     describe('option: maxAge', function () {
-      it('should error for claims issued before a certain timespan', function (done) {
-        clock = sinon.useFakeTimers(1437018582500);
-        var options = {algorithms: ['HS256'], maxAge: '321ms'};
 
-        jwt.verify(token, key, options, function (err, p) {
-          assert.equal(err.name, 'TokenExpiredError');
-          assert.equal(err.message, 'maxAge exceeded');
-          assert.equal(err.expiredAt.constructor.name, 'Date');
-          assert.equal(Number(err.expiredAt), 1437018582321);
-          assert.isUndefined(p);
-          done();
+      [String('3s'), '3s', 3].forEach(function(maxAge) {
+        it(`should error for claims issued before a certain timespan (${typeof maxAge} type)`, function (done) {
+          clock = sinon.useFakeTimers(1437018587000); // iat + 5s, exp - 5s
+          var options = {algorithms: ['HS256'], maxAge: maxAge};
+
+          jwt.verify(token, key, options, function (err, p) {
+            assert.equal(err.name, 'TokenExpiredError');
+            assert.equal(err.message, 'maxAge exceeded');
+            assert.equal(err.expiredAt.constructor.name, 'Date');
+            assert.equal(Number(err.expiredAt), 1437018585000);
+            assert.isUndefined(p);
+            done();
+          });
         });
       });
 
-      it('should not error for claims issued before a certain timespan but still inside clockTolerance timespan', function (done) {
-        clock = sinon.useFakeTimers(1437018582500);
-        var options = {algorithms: ['HS256'], maxAge: '321ms', clockTolerance: 100};
+      [String('6s'), '5s', 5].forEach(function (maxAge) {
```

Comment:
All the other multi-tests have different representations of the same time; this is different.  Seems like a mistake?

Reply:
Yep, copy&paste mistake. Fixing.